src/App.tsx

```diff
@@ -1,61 +1,134 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import { PreparedTodo } from './components/types/preparedTodo';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const preparedTodos: PreparedTodo[] = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId),
+  };
+});
+
+let maxTodoId = [...todosFromServer].sort((el1, el2) => (
+  el2.id - el1.id))[0].id;
+
+const addTodo = (title: string, userName: string) => {
+  maxTodoId += 1;
+
+  const newTodo = {
+    id: maxTodoId,
+    title,
+    completed: false,
+    userId: 1,
+    user: usersFromServer.find(user => user.name === userName),
+  };
+
+  preparedTodos.push(newTodo);
+};
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState('');
+  const [errorUser, setErrorUser] = useState(false);
+  const [errorTitle, setErrorTitle] = useState(false);
+
+  const handleChange = (
+    event: React.ChangeEvent<HTMLInputElement | HTMLSelectElement>,
+  ) => {
+    const { name, value } = event.target;
+
+    if (name === 'title') {
+      setTitle(value);
+    } else {
+      setUser(value);
+    }
```

Comment:
what if we will add more inputs?
it will be more `if else`?
better to use switch